6_pytest_output/README.md

```diff
@@ -75,9 +82,16 @@ performing the replacement.
 
 Overall we have
 ```
-vi(             select in parenthesis
+vip             select the paragraph
```

Comment:
Curious about this change.  I think `vip` will get everything in a block of text, e.g. between new lines.  `vi(` should be more specific

Reply:
I thought we wanted to read in the `:r output.tsv` and change the tabs to commas. After the `:r` command we have loaded in that block of text which we can select with `vip` before doing the substitute command. I don't see where the parenthesis are, maybe I'm not looking at the right text?

Reply:
I may also be thinking of something else... You're right here!